src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -71,6 +75,24 @@ export class HuiDialogEditCard extends LitElement {
     }
   }
 
+  protected updated(changedProps: PropertyValues): void {
+    if (
+      !this._cardConfig ||
+      this._documentationURL !== undefined ||
+      !changedProps.has("_cardConfig")
+    ) {
+      return;
+    }
+
+    const oldConfig = changedProps.get("_cardConfig") as LovelaceCardConfig;
+
+    if (oldConfig && this._cardConfig!.type === oldConfig.type) {
+      return;
+    }
+
+    this._documentationURL = getCardDocumentationURL(this._cardConfig!.type);
```

Comment:
```suggestion
    if (!oldConfig || this._cardConfig!.type !== oldConfig.type) {
	    this._documentationURL = getCardDocumentationURL(this._cardConfig!.type);
    }

```

Reply:
Damnit I had it this way but thought the other way was better..

Reply:
Updated that a bit as well to take advantage of `?` 

```ts
if (oldConfig?.type !== this._cardConfig!.type) {
    this._documentationURL = getCardDocumentationURL(this._cardConfig!.type);
}
```

Reply:
Use it only when you want to execute a large piece of code and otherwise nothing.

I a general `updated` function it will likely result in weird behavior if someone adds something on the bottom later.

Reply:
Makes sense. Thanks for the explanation